src/core/public/injected_metadata/injected_metadata_service.ts

```diff
@@ -146,6 +147,14 @@ export class InjectedMetadataService {
       getBranding: () => {
         return this.state.branding;
       },
+
+      getWazuhVersion: () => {
+        return this.state.wazuhVersion;
+      },
+
+      getWazuhDocVersion: () => {
+        return this.state.wazuhVersion.slice(0, 3) || 'current';
```

Comment:
**nitpick**: I would recommend another approach to get the value. This approach returns an unexpected value when the `major` or `minor` number has more than 1 digit, for example, `4.10` or `10.0`.

**suggestion**: split by `.` and join the first two elements.
```js
return this.state.wazuhVersion.split('.').slice(0, 2).join('.') || 'current';
```

This should avoid possible future problems while using the version format of `major.minor.patch`.